eox_tenant/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 Init for eox-tenant.
 """
-__version__ = '1.3.0'
+__version__ = '2.0.0'
```

Comment:
We are already in V2.0.0. I dont think this is necessary for the PR. We should leave the releases as part of a separate PR.

Reply:
@felipemontoya  Sorry, I forgot to change the base branch, this diff is not visible against the master branch